package.json

```diff
@@ -79,7 +79,7 @@
     "lerna": "^2.11.0",
     "link-parent-bin": "^0.2.0",
     "markdown-magic": "^0.1.25",
-    "nightwatch": "^0.9.19",
+    "nightwatch": "^1.0.19",
```

Comment:
can we remove this dependencies? I know its peer of terra-toolkit, however, we are not doing nightwatch testing in this repo.

Reply:
That works too: https://github.com/cerner/terra-clinical/pull/462/commits/6daea92574939c135969b1f0ccacf693608c6a33